core/src/main/java/org/apache/accumulo/core/client/mapreduce/AbstractInputFormat.java

```diff
@@ -426,6 +442,7 @@ protected static ClientConfiguration getClientConfiguration(JobContext context)
      *          the table name for which the scanner is configured
      * @since 1.6.0
      */
+    @Deprecated
     protected abstract void setupIterators(TaskAttemptContext context, Scanner scanner, String tableName, RangeInputSplit split);
```

Comment:
Yes, exactly. The only benefit would be to push up the call up the hierarchy and not have to bother with it in InputFormatBase and AccumuloMultiTableInputFormat.